src/kde/test/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+#--------------------------------------------*-cmake-*---------------------------------------------#
+#
+# file   kde/test/CMakeLists.txt
+# author Mathew Cleveland
+# date   Nov. 10th 2020
+# brief  CMake configuration for KDE tests
+# note   Copyright (C) 2018-2020 Triad National Security, LLC.
+#        All rights reserved.
+#--------------------------------------------------------------------------------------------------#
+cmake_minimum_required(VERSION 3.9.0)
+project( kde_test CXX )
+
+#--------------------------------------------------------------------------------------------------#
+# Source files
+#--------------------------------------------------------------------------------------------------#
+
+file( GLOB test_sources *.cc )
+
+#--------------------------------------------------------------------------------------------------#
+# Directories to search for include directives
+#--------------------------------------------------------------------------------------------------#
+
+include_directories(
```

Comment:
As before, please don't use this command. The `Lib_kde` should provide all of the later 2 directories automatically and you don't have any header files in the `test` directory, so it isn't needed either.

Reply:
Fixed here and in the template.